testResources/json/course.json

```diff
@@ -0,0 +1,58 @@
+{
+    "api_version": 7,
+    "course": {
+        "id": 3,
+        "name": "test-course",
+        "details_url": "https://example.com/staging/courses/3.json",
+        "unlock_url": "https://example.com/staging/courses/3/unlock.json",
+        "reviews_url": "https://example.com/staging/courses/3/reviews.json",
```

Comment:
While at it, we could drop the `staging` word from the urls, maybe just search and replace for all the json files.